pkg/kratos/service.go

```diff
@@ -183,12 +183,20 @@ func (s *Service) CheckAllowedProvider(ctx context.Context, loginFlow *kClient.L
 	return s.contains(allowedProviders, fmt.Sprintf("%v", provider)), nil
 }
 
+func (s *Service) GetClientName(loginFlow *kClient.LoginFlow) string {
```

Comment:
It's only used internally so u can keep it private and lowercase

not a blocker anyway

Reply:
I didn't realize you could still use unexported field in the same package. Thanks!
Done in new revision!